src/rpc/server.cpp

```diff
@@ -245,6 +245,11 @@ static UniValue uptime(const JSONRPCRequest& jsonRequest)
                 + HelpExampleRpc("uptime", "")
         );
 
+    const UniValue& dummy_value = jsonRequest.params[0];
+    if (!dummy_value.isNull()) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "No dummy first argument may be included.");
+    }
+
```

Comment:
Can be fixed instead by replacing `> 1` with `> 0` a few lines up?

Reply:
Ah, that was what I thought at first.